main.tex

```diff
@@ -1178,7 +1178,7 @@ \subsection{Future versions of the \astropypkg core and affiliated packages}
 will simplify the code base, and reduce the testing overhead for the package.
 \astropypkg version 3.0 is currently scheduled for January 2018.
 
-In the next major release after version 3.0, scheduled for the summer of
+In the next major release after version 3.0, scheduled for around June
```

Comment:
I deliberately didn't say the month to keep the paper accurate even in a case of a likely release delay.

Reply:
I think 'mid-2018' would be the most appropriate